src/load/azext_load/data_plane/load_test/commands.py

```diff
@@ -0,0 +1,35 @@
+from azure.cli.core.commands import CliCommandType
+
+admin_custom_sdk = CliCommandType(
+    operations_tmpl="azext_load.data_plane.load_test.custom#{}"
+)
+
+
+def load_test_commands(self, _):
+    with self.command_group(
+        "load test", custom_command_type=admin_custom_sdk, is_preview=True
+    ) as g:
+        g.custom_command("create", "create_test", supports_no_wait=True)
+        g.custom_command("update", "update_test", supports_no_wait=True)
+        g.custom_command("delete", "delete_test", confirmation=True)
+        g.custom_command("list", "list_tests")
+        g.custom_show_command("show", "get_test")
```

Comment:
Q: what is `custom_show_command` ?

Reply:
It takes care of exceptions if resource is missing

https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#register-commands